pkg/webhook/testdata/v1beta1_to_v1beta2/annotation_no_status_v1beta1.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: groupsnapshot.storage.k8s.io/v1beta1
+kind: VolumeGroupSnapshot
```

Comment:
Should the webhook convert VolumeGroupSnapshotContent instead???

Reply:
Yes!